src/Sylius/Bundle/ApiBundle/Serializer/ProductVariantNormalizer.php

```diff
@@ -72,7 +72,7 @@ public function normalize($object, $format = null, array $context = [])
         try {
             $data['price'] = $this->priceCalculator->calculate($object, ['channel' => $channel]);
             $data['originalPrice'] = $this->priceCalculator->calculateOriginal($object, ['channel' => $channel]);
-        } catch (ChannelNotFoundException $exception) {
+        } catch (ChannelNotFoundException) {
```

Comment:
Why? I would be for reverting this change

Reply:
I saw a similar example earlier in reviews, `$exception` variable isn't used in the corresponding block of code

Reply:
Change reverted